azerothcore/azerothcore-wotlk
# fix(DB/gameobject): Import Old World herbs from Vmangos

## Changes Proposed:
- What the title says, but it is very messy imo and I'm open to suggestions. I'm not well-versed in SQL so I ended up doing a rather primitive method, I think. 
- Separates Black Lotus from the master herb pool
- Does away with the Master Herb pools in general, replacing them with different pools for each type of herb
- Pools the previously-unpooled Alterac Mountains
- Replaces broken spawns with sniffed ones
- Removes another 391 unpooled spawns
- Removes some non-sensical spawns and updates some unreachable nodes.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7384
- Closes https://github.com/chromiecraft/chromiecraft/issues/1144
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6254
- Closes https://github.com/chromiecraft/chromiecraft/issues/794
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11268
- Closes https://github.com/chromiecraft/chromiecraft/issues/3192
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11551
- Closes https://github.com/chromiecraft/chromiecraft/issues/3423
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6778
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11397
- Closes https://github.com/chromiecraft/chromiecraft/issues/3191
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/10218
- Closes https://github.com/chromiecraft/chromiecraft/issues/2805
- May close a couple more that I didn't see

## SOURCE:

All data from Vmangos

## Tests Performed:

- Not tested beyond casual sightseeing

## How to Test the Changes:


1. See if any zones or herbs are missing
2. See if pools work
3. See if herbs respawn

Use the following query to see all spawns:
```
UPDATE `pool_template` SET `max_limit`=200 WHERE `entry` BETWEEN 489 AND 716;
```

## Known Issues and TODO List:


- [x] Remove old unpooled spawns.
- [ ] Review new spawns with weird coordinates such as the underwater Wild Steelbloom in Cragspool and unreachable Stranglekelp near Yojamba
- [x] Accidentally duplicated Stranglekelp spawns in Stranglethorn. Now there is an empty 680 entry that will haunt me forever.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
